core/templates/dev/head/components/HintAndSolutionButtonsDirective.js

```diff
@@ -63,7 +65,16 @@ oppia.directive('hintAndSolutionButtons', [
             return HintsAndSolutionManagerService.isSolutionViewable();
           };
 
+          $rootScope.$on('hintReleased', function() {
```

Comment:
Pull the string out into a named event and put it with the other oppia.constant()'s that represent events.